http/src/client/mod.rs

```diff
@@ -142,6 +142,11 @@ impl Client {
         self.state.default_allowed_mentions.clone()
     }
 
+    /// Get the Ratelimiter used by the client internally.
+    pub fn ratelimiter(&self) -> Option<Ratelimiter> {
+        self.state.ratelimiter.clone()
+    }
```

Comment:
Should probably have a note saying what it means if `None` is returned.

Reply:
Should be clarified now